lib/routes/install.js

```diff
@@ -67,7 +66,10 @@ function searchMatchingGenerators(app, term, cb) {
         return generatorMatchTerm(generator, term);
       }));
     }, cb);
-  });
+  }
+  got('yeoman.io/blacklist.json', {json: true})
```

Comment:
I wonder what's best practice now with got; `{json: true}` or doing it later with `response.json()` (just like browser `fetch()` API)